README.md

```diff
@@ -1,19 +1,22 @@
+<a name="readme-top"></a>
 
-
+<div align="center">
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
```

Comment:
- [ OPTIONAL ] Please remove the Microverse logo and this entire part of the README as it is currently irrelevant in this project. If you have your logo, you can use it :100: 
- Part to remove
![image](https://user-images.githubusercontent.com/102699958/227727411-0b098ee1-5d5b-4ff7-8849-4e6c899ebed0.png)